torch/quantization/fake_quantize.py

```diff
@@ -79,7 +79,12 @@ def forward(self, X):
         if self.observer_enabled:
             self.activation_post_process(X.detach())
             _scale, _zero_point = self.calculate_qparams()
-            self.scale, self.zero_point = _scale.to(self.scale.device), _zero_point.to(self.zero_point.device)
+            _scale, _zero_point = _scale.to(self.scale.device), _zero_point.to(self.zero_point.device)
+            self.scale.resize_(_scale.shape)
```

Comment:
Why do we have to resize before calling `copy_`

Reply:
this is because in the constructor the tensors are initialized to empty, without the knowledge of incoming shapes.  During the first iteration, the incoming shape has a different size from the current shape, so we have to resize to match because `copy_` requires equal size.  Before this diff, this was happening implicitly with direct assignment.